api/core/workflow/nodes/parameter_extractor/parameter_extractor_node.py

```diff
@@ -1,4 +1,5 @@
 import json
+import logging
```

Comment:
I think it's better to remove the logger here.

Reply:
Previously, ParameterExtractorNode execution exceptions occurred, but the reason was not well reflected in the interface (only "Extra data: line 1 column 96 (char 95)" was displayed). According to the test code, the ParameterExtractorNode execution is more inclined to return None than throwing an exception when parsing errors occur.

The reason for introducing logger is to record some parsing errors in the log for subsequent improvements.